src/main/create-operator-json/operators/pulse.js

```diff
@@ -8,5 +8,6 @@ import r6operators from "r6operators";
 let pulse = new Operator(r6operators.pulse, operatorId.pulse, "N/A");
 
 pulse.addCounterNode(operatorId.finka, counterType.soft, "Pulse's Heartbeat Sensor will detect Attackers boosted by Finka's Adrenal Surge for twice the amount of time.");
+pulse.addCounterNode(operatorId.maverick, counterType.hard, "Pulse's HeartBeat Sensor will detect Maverick when attempting to put holes through walls.");
```

Comment:
Would we be able to change this to, please:
Pulse's Heartbeat Sensor will detect Maverick's Breaching Torch when attempting to put holes in walls.

Reply:
Apologies for the delay, this has now been resolved.